servicetalk-loadbalancer-experimental/src/main/java/io/servicetalk/loadbalancer/DefaultRequestTracker.java

```diff
@@ -152,26 +150,26 @@ public final int score() {
         if (concurrentCount > 0 && concurrentStamp != Long.MIN_VALUE) {
             // If we have a request concurrent we should consider how long it has been concurrent so that sudden
             // interruptions don't have to wait for timeouts before our scores can be adjusted.
-            currentEWMA = max(currentEWMA, nanoToMillis(currentTimeNanos - concurrentStamp));
+            currentEWMA = Integer.max(currentEWMA, nanoToMillis(currentTimeNanos - concurrentStamp));
```

Comment:
This is an alias for `Math.max`. Consider changing to `Math.max` to reduce call stack and directly benefit from `@HotSpotIntrinsicCandidate`. Same for all other cases with min/max for Long/Integer

Reply:
I changed it on purpose so that it's explicit what the types are, but will admit I didn't consider performance implications. Considering it now, in my minds eye there is no better candidate for inlining than examples like this that just forward the call so I suspect there will be no real difference. That said, if you feel strongly I will this (and the others) change it back.